src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -2147,7 +2147,7 @@ Please specify one of them using the --source option to proceed.</value>
     <value>The source type is invalid</value>
   </data>
   <data name="APPINSTALLER_CLI_ERROR_PACKAGE_IS_BUNDLE" xml:space="preserve">
-    <value>The MSIX file is a bundle, not a package</value>
+    <value>The MSIX file is a bundle, not a package.</value>

```

Comment:
Seems like most other strings for error codes don't end with a period, but some do. Not sure which one is better, but if we're changing it we should make it consistent across the board